### PR TITLE
initial

### PR DESCRIPTION
**Pull request by kamsz**

### CODE REVIEW

File: systemd.service [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Spacelift worker
+
+[Service]
```
Comment (limakzi) [inline systemd.service:4]: I would create separate one-time service to pull tags-metadata, token and secrets.

**Review by limakzi — CHANGES_REQUESTED** [review]

File: systemd.service (hunk 1 above)
Comment (kamsz) [reply systemd.service:4]: token is passed as userdata into /etc/sysconfig/spacelift-launcher

Comment (kamsz) [reply systemd.service:4]: in terms of one-time service, let's add it in next iteration?

**Review by limakzi — APPROVED** [review]
LGTM.

**Pull request merged**